decidim-comments/app/models/decidim/comments/comment.rb

```diff
@@ -21,9 +21,12 @@ class Comment < ApplicationRecord
       validate :commentable_can_have_replies
       validates :depth, numericality: { greater_than_or_equal_to: 0 }
       validates :alignment, inclusion: { in: [0, 1, -1] }
+      validate :same_organization
 
       before_save :compute_depth
 
+      delegate :organization, to: :commentable
```

Comment:
I don't know, doesn't look that complicated to me and add another layer of data integrity.